sys/include/net/ng_ipv6/netif.h

```diff
@@ -84,6 +84,15 @@ extern "C" {
  */
 
 /**
+ * @{
+ * @name Flags for the interfaces
```

Comment:
enums for flags are ugly (yes I know I had used them in that context myself, but I changed my opinion on that).